docs/docs/ab_testing.md

```diff
@@ -0,0 +1,95 @@
+Pathfora supports A/B testing on a global level as well as on a targeted audience. You can select to split a pool of users to show some proportion of them a group of modules "A" and the other portion a group "B", (group being a collection of 0 or more modules).
```

Comment:
> You can select to split a pool of users to show some proportion of them a group of modules

This reads funny. I think it has to do with using "select".

>  (group being a collection of 0 or more modules)

I'm not sure this is necessary. I don't think it adds any clarity.